### PR TITLE
CSS cleanup + Minified CSS

### PR DESCRIPTION
**Pull request by chariot6**
Added a minified CSS
CSS cleanup (removed rules that were overwritten elsewhere etc.)

### CODE REVIEW

File: css/uploadfile.css [hunk 1]
```diff
@@ -1,146 +1,135 @@
...
 
 .ajax-file-upload-filesize {
-width: 50px;
-height: auto;
-margin: 0 5px 5px 0px;
-display: inline-block;
-vertical-align:middle;
+    width: 50px;
+    height: auto;
+    margin: 0 5px 5px ;
```
Comment (VipSaran) [inline css/uploadfile.css]: missing left margin

File: css/uploadfile.css [hunk 2]
```diff
@@ -1,146 +1,135 @@
...
 }
 
 .ajax-file-upload-filename {
-width: 300px;
-height: auto;
-margin: 0 5px 5px 0px;
-
+    width: 300px;
+    height: auto;
+    margin: 0 5px 5px 0
```
Comment (VipSaran) [inline css/uploadfile.css]: missing semicolon

File: css/uploadfile.css [hunk 3]
```diff
@@ -1,146 +1,135 @@
...
+    font-weight: normal;
+    padding: 4px 15px;
+    text-decoration: none;
+    cursor: pointer;
+    text-shadow: 0 1px 0 #5b8a3c;
+    vertical-align: top;
+    margin: 5px 10px 5px 0;
 }
+
 .ajax-file-upload {
```
Comment (VipSaran) [inline css/uploadfile.css]: not just a cleanup, but a change of look and feel

File: css/uploadfile.css (hunk 2 above)
Comment (chariot6) [inline css/uploadfile.css]: Semicolons are optional at the end of a block, but it is still bad practice to omit them. I'll fix that.

**Pull request closed without merging**